plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineProjectWizard.java

```diff
@@ -41,7 +41,7 @@
 public abstract class AppEngineProjectWizard extends Wizard implements INewWizard {
 
   private AppEngineWizardPage page = null;
-  private AppEngineProjectConfig config = new AppEngineProjectConfig();
+  protected AppEngineProjectConfig config = new AppEngineProjectConfig();
```

Comment:
`final` is possible. I feel OK without a getter: this is an `abstract` class, and only updating `config` are (one of) its subclasses.